external/runtime/runtime.depproj

```diff
@@ -11,6 +11,12 @@
   <PropertyGroup Condition="'$(TargetGroup)' == 'uapaot'">
     <ProjectJsonTemplate>$(MSBuildThisFileDirectory)NETNative/project.json.template</ProjectJsonTemplate>
     <NuGetDeploySourceItem>Reference</NuGetDeploySourceItem>
+    <SkipFilterTargetingPackResolvedNugetPackages>false</SkipFilterTargetingPackResolvedNugetPackages>
   </PropertyGroup>
+  <ItemGroup Condition="'$(TargetGroup)'=='uapaot'">
+    <TargetingPackReference Include="System.Private.CoreLib" />
```

Comment:
We are switching to a newer filtering mechanism see how we filter in netfx.depproj.